getting-started/vs-code-integration/convert-project-wizard.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Convert to Telerik Project
+page_title: Convert to Telerik Project - Visual Studio Code Integration
+description: Learn how to convert your existing Blazor app to use Telerik components in it with a few clicks through our Visual Studio Code Templates.
+slug: getting-started-vs-code-integration-convert-project
+position: 3
+---
+
+# Convert to Telerik Application
+
+This article demonstrates how to enable the Progress&reg; Telerik&reg; UI for Blazor components in your existing Blazor app with a few mouse clicks.
+
+To add the Telerik Components to your existing Blazor application, use the **Convert Command** of the Telerik UI for Blazor Visual Studio Code Extension.
+
+## Get the Extension
+
+@[template](/_contentTemplates/common/general-info.md#vs-code-x-download)
+
+
+## Convert your project
+
+1. Press `Ctrl`+`Shift`+`P` in Windows/Linux or `Cmd`+`Shift`+`P` on Mac to open VSCode's extension launcher
+
+![Launch Convert Command](images/launch-convert-command.png)
+
+1. Type/Select `Telerik UI for Blazor Convert Command: Launch` and press `Enter` to launch the convert command
+
+1. The conversion process will start
+
+1. The following notification will appear in the bottom right corner of the VS Code to indicate that the conversion has successfully passed.
+
+![Success Notification](images/successfully-converted-example.png)
+
+  * If the Telerik components are already enabled in the project you will see the following notification:
+
+![Already converted project Notification](images/already-converted-example.png)
+
+1. You now have enabled the Progress&reg; Telerik&reg; UI for Blazor components in your Blazor app!
```

Comment:
This is out of the main list flow because the lines above need to be indented. Alternatively, make this a `>tip` to congratulate people on success, it does not have to be a list item